napari/layers/surface/_surface_constants.py

```diff
@@ -0,0 +1,25 @@
+from enum import auto
+
+from ...utils.misc import StringEnum
+
+
+class Shading(StringEnum):
+    """Shading: Shading mode for the surface.
+
+    Selects a preset shading mode in vispy that determines how
+    color is computed in the scene.
+    See also: https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glShadeModel.xml
+            Shading.NONE
+                Computed color is interpreted as input color, unaffected by
+                lighting. Corresponds to shading='none'.
+            Shading.FLAT
+                Computed colours are the color at a specific vertex for each
+                primitive in the mesh. Corresponds to shading='flat'.
+            Shading.SMOOTH
+                Computed colors are interpolated between vertices for each
+                primitive in the mesh. Corresponds to shading='smooth'
+    """
+
+    NONE = auto()
+    FLAT = auto()
+    SMOOTH = auto()
```

Comment:
My suggestion is to include the i18n for this change so something like
```python
SHADING_TRANSLATION = {
    Shading.NONE: trans._("None"),
    Shading.FLAT: trans._("Flat"),
    Shading.SMOOTH: trans._("Smooth"),
}
```

And then use this to populate the combobox so that the displayed item is the translated one and the stored value is the actual enum value.

An example of this can be found here https://github.com/napari/napari/pull/2498/files#diff-7b43dacbdfe011eea412f799bc5117e8b5cf760ccc0f1024153dcc91f7ca825bR59